routing/routing_tests/index_graph_tools.cpp

```diff
@@ -12,6 +12,12 @@ namespace routing_test
 {
 using namespace routing;
 
+// RestrictionTest
+void RestrictionTest::SetStarter(FakeEnding const & start, FakeEnding const & finish)
+{
+  m_starter = MakeStarter(start, finish, *m_graph);
```

Comment:
It would be nice to CHECK here that m_graph is non-null.

Reply:
done